src/generate.jl

```diff
@@ -74,3 +76,27 @@ function entrypoint(pkg::String, dir; preview::Bool)
         )
     end
 end
+
+function gentestproject(pkg::String, dir; preview::Bool)
+    genfile(pkg, dir, "test/Project.toml"; preview=preview) do io
+        toml = Dict(
+            "deps" => Dict("Test" => "8dfed614-e22c-5e08-85e1-65c5234f0b40")
+            )
+        TOML.print(io, toml, sorted=true, by=key -> (Types.project_key_order(key), key))
+    end
+end
+
+function genruntests(pkg::String, dir; preview::Bool)
+    genfile(pkg, dir, "test/runtests.jl"; preview=preview) do io
+        print(io,
+           """
+            using $pkg
+            using Test
```

Comment:
Why not `using $pkg, Test`?

Reply:
No particular reason. The `PkgTemplates.jl` package generates a `runtests.jl` file with the `using` statements on two lines, so I did that for consistency. But I have no preference.